examples/deform/linearized_deform.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Simple example using the operators of linearized deformations."""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+
+import numpy as np
+import odl
+
+
+# Define a rectangle function
+def rectangle(x):
+    # Characteristic function of [-0.5, -0.25]x[-0.5, 0.0]
+    return (x[0] >= -0.5) & (x[0] <= -0.25) & (x[1] >= -0.5) & (x[1] <= 0.0)
+
+
+# Discrete reconstruction space: discretized functions on the rectangle
+# [-1, 1]^2 with 100 samples per dimension.
+discr_space = odl.uniform_discr([-1, -1], [1, 1], (100, 100), interp='linear')
+
+# The template is a 100 x 100 pixel image
+template = discr_space.element(rectangle)
+
+# Define a displacement field that shifts an image horizontally by hx
+# to the right and vertically by hy up. The it rotates clockwise by theta.
+hx = -0.5
+hy = -0.5
+theta = np.pi/4
+disp_func = [lambda x: (np.cos(theta) - 1) * x[0] - np.sin(theta) * x[1] + hx,
+             lambda x: np.sin(theta) * x[0] + (np.cos(theta) - 1) * x[1] + hy]
+
+# Define the deformation operator where template is fixed
+fixed_templ_op = odl.deform.LinDeformFixedTempl(template)
+
+# Discrete a displacement field space corresponding to the above
+# discrete reconstruction space
+pspace = fixed_templ_op.domain
```

Comment:
dont save variables like this, simply go for `fixed_templ_op.domain` in the rest of the code. That makes it easier to read.